core/App/ehsm_napi.cpp

```diff
@@ -1446,29 +1472,42 @@ char* NAPI_GenerateQuote(const char *challenge)
  *          }
  *      }
  */
-char* NAPI_VerifyQuote(const char *quote, const char *nonce){
+char* NAPI_VerifyQuote(const char *quote_base64, const char *nonce)
+{
```

Comment:
where check quote is base64 encoding? in node.js or add to this function?

Reply:
yes, checked in js layer